api/KeyboardEvent.json

```diff
@@ -715,7 +715,7 @@
             "status": {
               "experimental": false,
               "standard_track": false,
```

Comment:
suggestion: (again) since this is specified, let's reflect that in the `standard_track` value

```suggestion
              "standard_track": false,
```

Reply:
Don't see any changes here

Reply:
```suggestion
              "standard_track": true,
```